src/main/kotlin/com/example/model/Device.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.model
+
+import kotlinx.datetime.Instant
+import kotlinx.serialization.Serializable
+@Serializable
+
```

Comment:
```suggestion

@Serializable
```
Put the `@Serializable` next to the class that is becoming serializable (in this case Device)